omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -103,15 +103,135 @@ def elastic_search_major_version
         # Decorate any JSON parsing exception or numeric exceptions when accessing and parsing the version field.
         raise "Unable to parse elasticsearch response #{e}"
       end
-      raise "Unsupported elasticsearch version of #{version}. There is current support for the major versions of 2 and 5." if version != 5 && version != 2
-
+      raise "Unsupported elasticsearch version of #{version}. There is current support for the major versions of 2, 5 and 6." if version != 6 && version != 5 && version != 2
       version
     else
       # Elasticsearch is disabled - this configuration setting should never be used in erlang.
       0
     end
   end
 
+  def create_elasticsearch_index
```

Comment:
[nit] `create_` implies this function takes action, when really it returns the index definition. Maybe `es_index_definition` would be a better name?

Reply:
good point! done!